src/app/containers/MediaPlayer/index.jsx

```diff
@@ -101,7 +102,7 @@ const MediaPlayerContainer = ({
   );
 
   return (
-    <>
+    <Figure>
```

Comment:
Figure shouldn't wrap the `Metadata` component - only the media player and caption.
If you can move the `<Figure>` to one row down, that'd be perfect.

Reply:
Addressed and fixed.